pynxtools/dataconverter/readers/em/geometry/msmse_convention.py

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright The NOMAD Authors.
+#
+# This file is part of NOMAD. See https://nomad-lab.eu for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Conventions used in the EBSD communnity
+
+suggested by D. Rowenhorst et al. in DOI: 10.1088/0965-0393/23/8/083501.
+"""
```

Comment:
This looks like a mistaken newline character. You can remove this so it reads pretty for HTML as well.

Reply:
fixed in pynxtools-em